src/include/pbs_nodes.h

```diff
@@ -265,7 +266,7 @@ struct	pbsnode {
 	long			 nd_nsnfree;	/* number of VPs free */
 	long			 nd_ncpus;	/* number of phy cpus on node */
 	short			 nd_written;	/* written to nodes file */
-	unsigned short		 nd_state;	/* state of node */
+	unsigned long		 nd_state;	/* state of node */
```

Comment:
One test case could be to set a job into maintenance mode and restart server and check that the node is still marked under "maintenance"